lib/xapi-stdext-std/xstringext.ml

```diff
@@ -170,17 +170,17 @@ module String = struct include String
     if n > 0 then (
       let len_f = String.length f and len_t = String.length t in
       let new_len = String.length s + (n * len_t) - (n * len_f) in
-      let new_s = String.make new_len '\000' in
+      let new_s = Bytes.make new_len '\000' in
```

Comment:
Should this be `b`? The `s` in `new_s` encodes `string`.

Reply:
Indeed